registry/api/errcode/errors_test.go

```diff
@@ -109,28 +108,6 @@ func TestErrorsManagement(t *testing.T) {
 		t.Fatalf("unexpected json:\ngot:\n%q\n\nexpected:\n%q", string(p), expectedJSON)
 	}
 
-	// Now test the reverse
```

Comment:
I personally believe that it's not part of the behavior, unless you accept deserialized errors from the api as a request objects that this library transforms into errors.